psalm.xml

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0"?>
 <psalm
-    totallyTyped="false"
```

Comment:
[totallyTyped](https://psalm.dev/docs/running_psalm/configuration/#totallytyped) is deprecated and [`resolveFromConfigFile` by default is `true`](https://github.com/vimeo/psalm/blob/0d524ecd5b4bea45de1aae77dd1638d4ea03457d/config.xsd#L62).